api/payara-bom/pom.xml

```diff
@@ -695,15 +695,15 @@
                 <version>${yasson.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.glassfish</groupId>
+                <groupId>org.eclipse.parsson</groupId>
                 <artifactId>jakarta.json</artifactId>
-                <version>${jsonp.version}</version>
+                <version>${parsson.version}</version>
             </dependency>
 
             <dependency>
                 <groupId>org.glassfish</groupId>
                 <artifactId>jsonp-jaxrs</artifactId>
-                <version>${jsonp.version}</version>
+                <version>${jsonp-jaxrs.version}</version>
```

Comment:
This one is from old API. I think it should be replaced with

````
    <groupId>org.eclipse.parsson</groupId>
    <artifactId>parsson-media</artifactId>
````

For that we also need to upgrade to Jersey 3.1, that depends on same artifact (3.1.0-M5 was tagged recently).

Reply:
Ah no, new jersey already requires Servlet 6.0, which we don't have, so that will not be a solution.

Rather we need to work around the dependency of `jersey-media-json-processing` (https://github.com/eclipse-ee4j/jersey/blob/3.x/media/json-processing/pom.xml)

On the place where we add this dependency, exclude jsop-jaxrs and add parsson-media dependency.

Reply:
It appears that I was too optimistic about possible replacement here -- we cannot get parsson package in until we also upgrade Jersey, which will happen later.

I then suggest just to call this property something like `transitional.jsonp-jaxrs.version` so that we know that is something that needs to change before we finish Jakarta EE 10 migration.